spec/support/shared_examples/does_not_cleanup_empty_strings.rb

```diff
@@ -8,7 +8,7 @@
     @redises.set(key, '')
     expect do
       @redises.send(method, *args)
-    end.to raise_error(defined?(default_error) ? default_error : RuntimeError)
+    end.to raise_error(defined?(default_error) ? default_error : Redis::BaseError)
```

Comment:
the version 5 BaseError now inherits from `StandardError` instead of `RuntimeError`, it felt more appropriate to narrow the scope of this test to `Redis::BaseError`.  This prevents needing to test the version and also limits the tests from passing on a different `StandardError` or `RuntimeError`.